### DISCUSSION TITLE
How can I remove the arrows in the QTabs component on the desktop?

### DISCUSSION THREAD

**Discussion by KINGER404** [quasarframework/quasar#12478] [General - Components / Directives / etc] [answered]
Is it possible to somehow disable the arrows in QTabs on the desktop?

If not, is it possible to add such a props?

**Comment by pikil** [comment, marked as answer]
That's interesting, but if you set ```align="right"``` on q-tabs, the arrows should dissappear (most likely would be fixed, but you can try to check it now)
![Screen Shot 2022-02-16 at 4 59 39 PM](https://user-images.githubusercontent.com/26779040/154193696-c3be7edb-e1a8-4abe-a54d-0282fbba78d8.png)

**Reply by KINGER404** [reply]
Hah, It works)
Thank you!
